com.gitlab.KRypt0n_.an-anime-game-launcher.yml

```diff
@@ -0,0 +1,109 @@
+app-id: com.gitlab.KRypt0n_.an-anime-game-launcher
+runtime: org.gnome.Platform
```

Comment:
do you really need the gnome runtime?

Reply:
The launcher uses [Neutralino](https://neutralino.js.org/), which uses GTK on Linux. I thought this is easier than adding modules to compile GTK

Reply:
gtk3 is in the freedesktop runtime

Reply:
didn't know that, I'll change it then

Reply:
Nope, that doesn't work, WebKitGTK isn't included in freedesktop, and I'm not compiling WebKit

Reply:
you said "gtk" not webkitgtk. :-)